packages/blockchain-api/README.md

```diff
@@ -24,7 +24,7 @@ Build and start:
 Deploy your app. The project will be built automatically by Google Cloud Build:
 
     using celotool:
-        yarn run cli deploy upgrade blockchain-api --config CONFIG_FILE --verbose --celo-env CELO_ENV
+        celotool deploy upgrade blockchain-api --config CONFIG_FILE --verbose --celo-env CELO_ENV
```

Comment:
Docs refers to it as `celotooljs` https://github.com/celo-org/celo-monorepo/blame/1a6a140dbba4c88a0ef25f752e4ba09e83bc1978/packages/celotool/README.md#L18

Reply:
We should change the docs to make it just celotool now that celotooljs is the only celotool

Reply:
I will update it in a follow-up PR